dcor/_energy.py

```diff
@@ -22,13 +22,23 @@ def _check_valid_energy_exponent(exponent):
 
 
 def _energy_distance_from_distance_matrices(
-        distance_xx, distance_yy, distance_xy):
-    """Compute energy distance with precalculated distance matrices."""
-    return (2 * np.mean(distance_xy) - np.mean(distance_xx) -
-            np.mean(distance_yy))
+        distance_xx, distance_yy, distance_xy, average=None):
+    """
+    Compute energy distance with precalculated distance matrices.
+
+    Parameters
+    ----------
+    average: Callable[[ArrayLike], float]
```

Comment:
This function is private so the parameter doc should really be added to `energy_distance` and `energy_test`.